pkg/generators/models/testdata/cases/allOf_enum_merging_and_validation/expected/model_connection.go

```diff
@@ -47,7 +47,7 @@ func (m Connection) Validate() error {
 			m.Properties, validation.NotNil,
 		),
 		"technology": validation.Validate(
-			m.Technology, validation.NotNil,
+			m.Technology, validation.Required,
```

Comment:
hmm i just realized that these changed ... i need to double check that

Reply:
I think that is the correct value given: https://github.com/contiamo/openapi-generator-go/pull/95/files/2e92689f7076065c27e9340b14de37bb780bfce9#diff-f6646d69f3c9d3794d9fde9eec4b73f1703ac88997bc53e93232810b6a5cd7f1L15